templates/table.js.tera

```diff
@@ -182,7 +182,7 @@ $(document).ready(function() {
                let max = Math.max(...values);
                if (brush_domains[title] != undefined) {
                    min = brush_domains[title][0];
-                   max = brush_domains[title][1];
+                   max = brush_domains[title][brush_domains[title].length - 1];
```

Comment:
I think we should calculate min and max from the whole array. As it can also be that a user specifies the domain in inverse order (or unsorted).

Reply:
Yep makes sense!